CHANGELOG.md

```diff
@@ -5,8 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [1.9.13] - 2024 05-29
```

Comment:
This release and [my other PR](https://github.com/nasa/gesdisc-meditor/pull/55) are going to conflict here. I'll fix this based on which gets approved first.